dynagrpc/__init__.py

```diff
@@ -21,10 +21,8 @@
 
 from . import _astcast
 
-try:
-    from typing import Literal
-except ImportError:  # TODO: Drop Python 3.7 compatibility
-    from typing_extensions import Literal
+
+from typing import Literal
```

Comment:
This would be better just before `import warnings`, keeping the alphabetical order in a block of standard libraries being imported. But this is not a big deal, this can be changed later in case we apply `isort` or add a configuration for it.